go.mod

```diff
@@ -156,7 +156,7 @@ require (
 require (
 	cloud.google.com/go/secretmanager v1.4.0
 	dubbo.apache.org/dubbo-go/v3 v3.0.3-0.20220610080020-48691a404537
-	github.com/Azure/azure-sdk-for-go/sdk/data/azappconfig v0.4.0
+	github.com/Azure/azure-sdk-for-go/sdk/data/azappconfig v0.4.1-0.20220914204640-4d445978fe75
```

Comment:
We should ask the SDK team to release 0.4.1 or 0.5.0. We should not pin a random commit, only tags.

We still have some time, please reach out to the team.

Reply:
Discussed in team chat - apparently the SDK team suggest using a direct commit in this case.